src/prophecy/layer/dense_layer.hh

```diff
@@ -55,21 +55,24 @@ void DenseLayer<T>::compile(std::weak_ptr<Layer<T>>   prev,
                                        prev.lock()->get_out_shape()[0]};
     if (init_ == "xavier")
     {
-        const T weights_std = sqrt(((T)2) / (w_shape.front(), w_shape.back()));
-        this->weights_              = xt::random::randn<T>(w_shape, 0,
-                                                           weights_std);
+        const T weights_std = sqrt(static_cast<T>(2)) / (w_shape.front() + w_shape.back()));
```

Comment:
It does not compile: the closing parenthesis of sqrt is inserted too early.

Reply:
Sorry for this typo, the code should work now